client/reader/stats.js

```diff
@@ -96,6 +96,11 @@ export function recordTrack( eventName, eventProperties ) {
 		eventProperties = Object.assign( { subscription_count: subCount }, eventProperties );
 	}
 
+	if ( location === 'topic_page' ) {
+		const tag = window.location.pathname.split( '/tag/' ).pop();
+		eventProperties = Object.assign( { tag: tag }, eventProperties );
```

Comment:
this will allow the eventProperties to overwrite the tag prop if it contains one. Is that what you want?